pillarbox-core-business/src/test/java/ch/srgssr/pillarbox/core/business/tracker/comscore/ComScoreTrackerIntegrationTest.kt

```diff
@@ -532,6 +537,32 @@ class ComScoreTrackerIntegrationTest {
 
         TestPlayerRunHelper.runUntilPlaybackState(player, Player.STATE_IDLE)
 
+        verifyOrder {
+            verifyPlayerInformation()
+            verifyCreatePlaybackSession()
+            verifyMetadata()
+            verifyPlaybackRate(playbackRate = 1f)
+            verifyBufferEvents()
+            verifySeekEvent(0L)
+            verifyPlayEvent()
+            verifyPauseEvent()
+        }
+        confirmVerified(streamingAnalytics)
+    }
+
+    @Test
+    fun `player prepared, playing and released`() {
+        player.setMediaItem(SRGMediaItemBuilder(URN_NOT_LIVE_VIDEO).build())
+        player.prepare()
+        player.playWhenReady = true
+
+        TestPlayerRunHelper.runUntilPlaybackState(player, Player.STATE_READY)
+
+        clock.advanceTime(2.minutes.inWholeMilliseconds)
+        player.release()
+
+        // TestPlayerRunHelper.runUntilPlaybackState(player, Player.STATE_IDLE)
```

Comment:
Yes because release doesn't move the state to Player.STATE_IDLE we could replace it by waiting until `player.isReleased` is true. But it is not needed.